lib/config/defaultTsConfig.ts

```diff
@@ -132,15 +145,15 @@ module.exports = {
       profiles: {
         admin: {
           rateLimit: 0,
-          policies: [ { roleId: 'admin'} ]
+          policies: [ { roleID: 'admin'} ]
```

Comment:
This property should stay `roleId` to match Kuzzle internal data representation. This comment apply for others occurrences